src/platform/silabs/Logging.cpp

```diff
@@ -131,14 +137,23 @@ static void PrintLog(const char * msg)
     {
         size_t sz;
         sz = strlen(msg);
+#if SILABS_LOG_OUT_UART
+        uartLogWrite(msg, sz);
+#endif
```

Comment:
since log_uart and pw_rpc_enable but use the uart for logs I would suggest a 
```
#if SILABS_LOG_OUT_UART
   uartLogWrite(msg, sz);
#elif PW_RPC_ENABLED
    PigweedLogger::putString(msg, sz);
#endif  
```

In same order of ideas. I see you define `SILABS_LOG_OUT_RTT` to 0 when defining `SILABS_LOG_OUT_UART` but maybe a else of elif for the `SEGGER_RTT_WriteNoLock` would be more explicit that both aren't used at the same time.